src/Simulation/Native/src/simulator/factory.cpp

```diff
@@ -63,7 +63,7 @@ MICROSOFT_QUANTUM_DECL unsigned create(unsigned maxlocal)
 {
     std::lock_guard<std::shared_mutex> lock(_mutex);
 
-    size_t emptySlot = -1;
+    size_t emptySlot = (size_t)-1;
```

Comment:
Doesn't matter much, but probably static_cast<>?

Reply:
I assume that the C-style casts are equivalent to `reinterpret_cast<>()`, and at least when applied to primitive types I personally prefer the C-style casts for brevity.
If nobody has strong objections I would prefer to leave as is. Otherwise no problem on my end to change, just let me know.